### PR TITLE
Increase max URI length to 4k

### PR DESCRIPTION
**Pull request by helpermethod**
Doubles the max URI length (currently 2k).

Related to https://sdkman.slack.com/archives/CJTNQA94M/p1639954703084700.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -64,3 +64,7 @@ releaseProcess := Seq[ReleaseStep](
   commitNextVersion,
   pushChanges
 )
+
+PlayKeys.devSettings := Seq(
```
Comment (marc0der) [inline build.sbt:68]: Just wondering, but is this really required for the runtime behaviour of the app? I'd think the setting in `application.conf` would be enough.

**Review by marc0der — APPROVED** [review]

File: build.sbt (hunk 1 above)
Comment (helpermethod) [reply build.sbt:68]: This was also confusing for me. It seems in dev mode certain settings can't be configured via `application.conf`.

https://www.playframework.com/documentation/2.6.x/ConfigFile#Using-with-the-run-command

https://github.com/playframework/playframework/issues/7617

**Comment by helpermethod** [comment]
I've tested the change both in dev mode and by running the Docker image and everything runs fine.

File: build.sbt (hunk 1 above)
Comment (marc0der) [reply build.sbt:68]: Since this is an extreme edge case, I wouldn't bother with it in dev mode. Let's drop this snippet for now.

**Comment by marc0der** [comment]
Excellent work, thanks for taking the time to delve into this @helpermethod. I'll release this today.

**Pull request merged**